etcddb/etcddb_test.go

```diff
@@ -2,14 +2,31 @@ package etcddb
 
 import (
 	"fmt"
+	"os"
 	"testing"
 
+	"github.com/smartystreets/gunit"
```

Comment:
I would suggest using ```github.com/stretchr/testify/suite``` to not introduce new library.

Reply:
Updated the test to use testify/suite instead of gunit